docs/manual/docs/user-guide/harvesting/harvesting-geonetwork-20.md

```diff
@@ -0,0 +1,27 @@
+# GeoNetwork 2.0 Harvester {#gn2_harvester}
+
+GeoNetwork 2.1 introduced a new powerful harvesting engine which is not compatible with GeoNetwork version 2.0 based catalogues. Old 2.0 servers can still harvest from 2.1 servers but harvesting metadata from a v2.0 server requires this harvesting type. Due to the fact that GeoNetwork 2.0 was released more than 5 years ago, this harvesting type is deprecated.
```

Comment:
I am a little confused by this sentence due - I expect there is a difference between 2.0 and 2.1 API which you are describing?  There is no need to describe the capabilities of a 2.1 server in 4.2.x docs ...

It is not clear to me when support for 2.0 ended, more than 5 years ago is ... 2018?

```suggestion
This harvester is required for working with GeoNetwork 2.0. GeoNetwork 2.1 introduced a new powerful harvesting engine which is not compatible with the previous GeoNetwork version 2.0 based catalogues. 

This harvesting type is deprecated as GeoNetwork 2.0 has reached end-of-life.


| Harvester                | Harvet from                          |
| ------------------------ | ------------------------------------ |
| GeoNetwork 2.0 Harvester | GeoNetwork 2.0 Catalogue             |
| GeoNetwork 3.x Harvester | GeoNetwork 2.1 and greater           |
| GeoNetwork 3.x Harvester | GeoNetwork 3.x series                |
| GeoNetwork 4.x Harvester | GeoNetwork 4.x series                |
```